HIRING.md

```diff
@@ -43,7 +43,7 @@ We’re obviously trying to assess the candidate but remember they are assessing
 
 ## Technical Questions
 - What's the superclass of UITableView?
-- Explain how he or she would achieve smooth scrolling in a collection view of 100 000 images
+- Have you faced any challenges regarding achieving 60 frames per second in iOS? Tell us more about it
```

Comment:
Good point, let's only ask stuff that could be relevant in a daily basis, nobody cares if you know what's the super class of UITableView :D